src/fragmentarium/domain/pager.ts

```diff
@@ -0,0 +1,12 @@
+export interface NextAndPrevFragment {
```

Comment:
There is no need to shorten words, i.e. "previous" should be preferred over "prev".

Maybe `FragmentPagerData` and `FolioPagerData` are better names`?